people-details-sql-jdbc/src/main/java/connector/SQLConnector.java

```diff
@@ -0,0 +1,43 @@
+package connector;
+
+import connector.enums.Gender;
+import connector.enums.HighestEducation;
+
+import java.sql.*;
+
+public record SQLConnector(String url, String username, String password) {
+    public void getConnection() {
+        try (Connection connection = DriverManager.getConnection(url, username, password)) {
+            PreparedStatement ps = connection.prepareStatement("SELECT * FROM person");
+            ResultSet rs = ps.executeQuery();
+
+            while (rs.next()) {
+                int id = rs.getInt("id");
+                String firstName = rs.getString("first_name");
+                String lastName = rs.getString("last_name");
+                String motherName = rs.getString("mother_name");
+                String fatherName = rs.getString("father_name");
+                Date birthDate = rs.getDate("birth_date");
```

Comment:
please convert to `LocalDate` on the Java side - remember the advantages of the Java8 date-time API